Dockerfile.testing

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
+USER root
+
+WORKDIR /cvdi-stream
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Add build tools.
+RUN apt-get update && apt-get install -y software-properties-common wget git make gcc-7 g++-7 gcc-7-base && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 100 && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-7 100
+
+# Install cmake.
+RUN apt install -y libprotobuf-dev protobuf-compiler
+RUN apt install -y cmake
+
+# Install librdkafka.
+RUN apt-get install -y libsasl2-dev libsasl2-modules libssl-dev librdkafka-dev
+
+# install python for testing
+RUN apt-get install -y python3
+
+# add the source and build files
+ADD CMakeLists.txt /cvdi-stream
+ADD ./src /cvdi-stream/src
+ADD ./cv-lib /cvdi-stream/cv-lib
+ADD ./include /cvdi-stream/include
+ADD ./kafka-test /cvdi-stream/kafka-test
+ADD ./unit-test-data /cvdi-stream/unit-test-data
+ADD ./config /cvdi-stream/config
+
+# Do the build.
+RUN export LD_LIBRARY_PATH=/usr/local/lib && mkdir /cvdi-stream-build && cd /cvdi-stream-build && cmake /cvdi-stream && make
```

Comment:
Is this /cvdi-stream-build folder or a build folder inside /cvdi-stream? like /cvdi-stream/build ```mkdir -p /cvdi-stream/build && cd /cvdi-stream/build && cmake /cvdi-stream```

Reply:
Can we reuse the ./devcontainer and replace the Dockerfile.testing with the vscode test task for the testing?

Reply:
Project files are stored in `/cvdi-stream` and are used to build executables in `/cvdi-stream-build`.

Reply:
The `Dockerfile.testing` file is used by scripts involved in the execution of the `do_kafka_test.sh` script, which is an integration test between the PPM & Kafka. The primary difference is that python is included so that the 'test_in.py' and 'test_out.py' scripts are usable. I don't think this should be replaced by using the dev container.